functions/shiritori/index.js

```diff
@@ -98,29 +98,29 @@ exports.kanas = (word) => new Set((() => {
 })())
 
 // しりとりのゲームループ。
-exports.interact = (dict, word, previousInputs, callbacks) => {
-  const next = exports.kanas(word)
-  if ((next.size === 0) || !exports.check(word, previousInputs)) {
-    return callbacks.lose()
+exports.interact = (dict, word, chain) => new Promise((resolve, reject) => {
+  const next = exports.kanas(word);
```

Comment:
Could you remove unnecessary `;` character?

Reply:
Done